webapp/tests/forms/steps/lotse/test_has_disability.py

```diff
@@ -96,4 +96,57 @@ def test_if_person_b_has_disability_set_yes_then_do_not_raise_validation_error(s
         try:
             HasDisabilityPersonBPrecondition.parse_obj(data)
         except ValidationError:
-            pytest.fail("Should not raise a validation error")
\ No newline at end of file
+            pytest.fail("Should not raise a validation error")
+
+
+class TestStepDisabilityPersonAValidation:
+    def test_if_required_value_is_given_then_validation_should_be_success(self, new_test_request_context):
+        data = MultiDict({'person_a_has_disability': 'yes'})
+        with new_test_request_context(form_data=data):
+            step = LotseStepChooser().get_correct_step(
+                StepDisabilityPersonA.name, True, ImmutableMultiDict(data))
+            form = step.render_info.form
+            assert form.validate() is True
+
+    def test_if_required_value_is_not_give_validation_should_be_failure(self, new_test_request_context):
```

Comment:
Maybe we could rename it to:
```suggestion
    def test_if_required_value_is_not_given_then_validation_should_be_failure(self, new_test_request_context):
```